env0/resource_environment.go

```diff
@@ -799,6 +804,11 @@ func getCreatePayload(d *schema.ResourceData, apiClient client.ApiClientInterfac
 		payload.PullRequestPlanDeployments = boolPtr(val.(bool))
 	}
 
+	//lint:ignore SA1019 reason: https://github.com/hashicorp/terraform-plugin-sdk/issues/817
+	if val, exists := d.GetOkExists("prevent_auto_deploy"); exists {
```

Comment:
you're missing more use-cases.

Search for the string "auto_deploy_on_path_changes_only"

Reply:
why tho? I'm interested only in `Create` flow - update and delete are not interesting to me (even if this property is changed - nothing should be sone because its not really persisted at the backend)